languages/go/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,99 @@
+## Packages
+
+In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package is preferred to be the same as the folder it is located in.
+
+```go
+package greeting
+```
+
+Standard library packages are imported using their name. Other packages are importet using the url of the package's location. A package can be given an alias when importing. This can be used to avoid package name conflicts:
+
+```go
+package greeting
+
+import (
+	"errors"
+
+	errs "github.com/pkg/errors"
+)
+```
+
+An importet package is then addressed with the package name or alias:
+
+```go
+// using the internal errors package
+errors.New("Connection not established")
+
+// using the errors package located at github.com/pkg/errors by alias
+errs.New("Connection not established")
+```
+
+## Integers
+
+Go is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by assigning a value to have the Go compiler infer its type based on the assigned value.
+
+```go
+var explicitVar int // Explicitly typed
+implicitVar := 10   // Implicitly typed
+```
+
+The value of a variable can be assigned using the [`:=` operator][assignment] and updated using the [`=` operator][assignment]. Once defined, a variable's type can never change.
+
+```go
+count := 1 // Assign initial value
+count = 2  // Update to new value
+
+// Compiler error when assigning different type
+// count = false
+```
+
+Integer values are defined as one or more (consecutive) digits and support the [default mathematical operators][operators].
+
+## Functions
+
+Functions in Go are considered [first class citizens][first-class-functions] making them very powerful. They can be used as:
+- [public and private][public-vs-private] functions of a package
+- attached to custom types also called a [method][methods] ([public and private][public-vs-private])
+- assigned to variables
+- passed into to functions as parameters and returned from functions (higher order functions)
+- create custom types
+- be used anonymously
+- be used as closures
+
+TODO: is above OK or too much? Details will be in later exercises...
```

Comment:
I'm not sure actually. I like that you stress that functions are very important in Go. I'm wondering if students should know about first-class functions at this point, or if it is just enough for them to know that functions are the fundamental building block for creating functionality in a Go application. @iHiD any thoughts?

Reply:
Note: We could move it to the functions exercise, but the functions exercise will not handle the entire list mentioned here either... And I like the complete list to be somewhere.

Still would like to know your opinion @iHiD as this is the first exercise after all.

Reply:
I'm _think_ I'm ok with "too much" info being in the `after.md`. We just need to make it clear in the UI that the things in here will be expounded in future exercises. But that's a UI problem, not a problem here. 

For now, I'd say merge (and this has literally just been merged as I'm typing so :+1:) and we can always do another pass later down the line once lots of exercises are in place.